Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex

```diff
@@ -149,17 +192,31 @@ \subsection{Base Wheel}
   \label{wheel step 2}
 \end{figure}
 
-With that the base wheel assembly should be done! Repeat the above three steps for all six of the wheels. 
+With that the base wheel assembly should be done! Repeat the above three steps for all six of the wheels.
 
 \end{enumerate}
 
 \subsection{Corner Wheels}
 Now, we need to create the four corner wheels. These build on what we made for the base wheel assemblies in the last step. For the corner motors, we will need to extend the attachment point upwards as it is important that the axes of the corner steering motors are directly above the middle of the wheels to enable smooth steering.
 
-\begin{figure}[H]
-	\centering
-	\includegraphics[width=1\textwidth]{"Pictures/Wheels/Corner Wheel Parts".PNG}
-\end{figure}
+\begin{table}[H]
+    \centering
+    \arrayrulecolor{lightgray}
+    \sffamily\footnotesize
+    \captionsetup{font={sf,bf}}
+    \caption{Parts/Tools Necessary}
+    \begin{tabular}{|N|Q|Q|I|N|Q|Q|I|}
+        \hline
+        \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} & \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} \\
+        \hline
+        3" Channel & S2 & 4 & \partimg{../../../images/parts_list/S2.jpg} & \#6-32 x 1/4" Screw & B1 & 76 & \partimg{../../../images/parts_list/B1.png} \\ \hline
+        4.5" Channel & S4 & 4 & \partimg{../../../images/parts_list/S4.jpg} & \#6-32 x 1/2" Button Head Screw & B3 & 8 & \partimg{../../../images/parts_list/B3.png} \\ \hline
+        Channel Connector Plate & S6 & 16 & \partimg{../../../images/parts_list/S6.jpg} & \#6-32 x 3/8" Threaded Standoff & T3 & 4 & \partimg{../../../images/parts_list/T3.jpg} \\ \hline
+        Channel Connector Plate & S12 & 4 & \partimg{../../../images/parts_list/S12.jpg} & Allen Key Set & & & \partimg{../../../images/parts_list/D2.jpeg} \\ \hline
+        Plain Bore Gear & S26 & 4 & \partimg{../../../images/parts_list/S26.jpg} & 5/16 Wrench & & & \partimg{../../../images/parts_list/D1.jpg} \\ \hline
+        Loctite & S34 & & \partimg{../../../images/parts_list/S34.jpg} & & & & \\ \hline
```

Comment:
Name should be 'Loctite Threadlocker', not 'Loctite'.  Ref should be S36, not S34.

Reply:
Same issue as the other. This is S34 in the master parts list and will update the instructions to match.